src/Styleguide/Pages/Artwork/Components/__stories__/ArtworkSidebar/ArtworkSidebarCommercial.story.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react"
+import { storiesOf } from "storybook/storiesOf"
+import { ArtworkSidebarCommercial as Commercial } from "Styleguide/Pages/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial"
+import {
+  ContactForPriceWork,
+  FoSaleArtworkNoEditions,
+  FoSaleArtworkWithMultipleEditions,
+  FoSaleArtworkWithOneEdition,
+} from "Styleguide/Pages/Fixtures/Artwork/Sidebar/Commercial"
+import { Section } from "Styleguide/Utils/Section"
+import { RelayStubProvider } from "Utils/RelayStubProvider"
+
+storiesOf("Styleguide/Artwork/Sidebar", module)
+  .addDecorator(story => <RelayStubProvider>{story()}</RelayStubProvider>)
```

Comment:
cc  @oxaudo @zephraph @mzikherman this StubProvider component mocks the relay environment so that you can pass static fixture data down into the component without breaking errors.

Reply:
👍